Header header
uint8 id
geometry_msgs/Point32 comPoints
geometry_msgs/Point32 headPoint
std_msgs/Int32 distanceThreshold
